### DISCUSSION TITLE
ArduinoDroid IDE - does not have sys/types.h file

### DISCUSSION THREAD

**Discussion by DrDan666** [olikraus/u8g2#1799] [Q&A]
Every example fails to compile… missing sys/types.h file. Here is one compile result...

/sdk/hardware/tools/avr/bin/avr-gcc -no-canonical-prefixes -c -g -Os -w -mmcu=atmega328p -DF_CPU=16000000L -DUSB_VID=null -DUSB_PID=null -DARDUINO=10812 -DIDE="ArduinoDroid" -DARDUINO_ARCH_AVR -DARDUINO_AVR_UNO /U8g2/src/clib/mui_u8g2.c -o /librariesBuild/U8g2/clib_mui_u8g2.c.o In file included from /U8g2/src/clib/mui_u8g2.c:95:0:
/U8g2/src/clib/mui.h:45:23: fatal error: sys/types.h: No such file or directory
 #include <sys/types.h>

I can run the 128x64 mono I2C using other libraries. Sure hope you can help.

**Comment by olikraus** [comment]
The include line can be removed from mui.h
I will fix this in the repo. Thanks.

**Comment by DrDan666** [comment]
Thanks a million. Works perfectly!
Best regards,
Dan

⁣​

On Mar 1, 2022, 11:09 AM, at 11:09 AM, olikraus ***@***.***> wrote:
>The include line can be removed from mui.h
>I will fix this in the repo. Thanks.
>
>-- 
>Reply to this email directly or view it on GitHub:
>https://github.com/olikraus/u8g2/discussions/1799#discussioncomment-2274966
>You are receiving this because you authored the thread.
>
>Message ID:
***@***.***>

**Reply by Dirk-St** [reply]
Mit Gruss

Auch von mir vielen Dank.
Ich hatte das gleiche Problem
mit sys/types.h.
Dirk

**Comment by olikraus** [comment]
It should be fixed in the next release
